docs/use/dapp-staking/for-stakers/move-staked-tokens.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 3
+---
+
+import Figure from "/src/components/figure"
+
+# Move staked tokens between dApps
+
+Once your tokens have been staked on a dApp, you always have the option of reconsidering your decisions and moving them to another dApp.
+
+1) Go to the **Staking Panel** in the [Asset Page](https://portal.astar.network/astar/assets);
+2) Under **My dApps**, click on the `Move` button of the dApp from which you want to move your tokens.
+
+<Figure src={require('/docs/use/dapp-staking/for-stakers/img/MydApps_Panel_1.png').default } width="75%" /> 
+
+3) Select the dApp(s) to which you want to `Move` your tokens and enter the desired amount;
+
+<Figure src={require('/docs/use/dapp-staking/for-stakers/img/Staking_Page_3.png').default } width="75%" /> 
+
+4) Confirm your decisions by clicking on the **Confirm Button** and sign the transaction on the network;
+
+Bear in mind that when you move tokens from one dApp to another, you are `unstaking` from one dApp to `stake` on a new dApp, which therefore has an impact on your rewards for those dApps:
+
+- If you move tokens from a dApp and your staked tokens are less than the [minimum staking](/docs/use/dapp-staking/for-stakers/#parameters) amount for a dApp, all your tokens will be `unstaked` from that dApp;
+- If you move your tokens from one dApp to another during an `Era` , you'll lose the **basic staking rewards** for that `Era` for the dApp you moved, and you won't earn rewards on the new dApp until the next `Era`;
```

Comment:
Maybe to rewrite this sentence since tokens will be always moved during some era. Something like
If you move your tokens from one dApp to another one, you'll lose the **basic staking rewards** for the current Era for the dApp you moved from, and you won't earn rewards on the new dApp until the next `Era`;